### DISCUSSION TITLE
Image Manipulation

### DISCUSSION THREAD

**Discussion by nikhil-lu210** [spatie/laravel-medialibrary#2026] [Ideas] [answered]
### **You should add this on your documentation**
**We faced a problem that doesn't allow us to fill/fill the images with its original needed. So here we found the solution**

`->fit('fill', 440, 580);`

The types are: `contain`, `max`, `fill`, `stretch`, `crop`

**Comment by freekmurze** [comment, marked as answer]
The documentation of the image options is here: https://spatie.be/docs/image/v1/introduction

Feel free to PR improvements.

**Reply by nikhil-lu210** [reply]
Thank You.
I also found the documentation .

It's really very helpful.
